flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/main/java/com/ververica/cdc/connectors/values/source/ValuesDataSource.java

```diff
@@ -112,7 +112,7 @@ public ValuesSource(
 
         @Override
         public Boundedness getBoundedness() {
-            return Boundedness.BOUNDED;
+            return isInSnapshotPhase ? Boundedness.BOUNDED : Boundedness.CONTINUOUS_UNBOUNDED;
```

Comment:
Need to add an option for Source to control whether to support Boundedness.CONTINUOUS_UNBOUNDED, will do later.